src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/Validators/ValidationBehavior.shared.cs

```diff
@@ -197,20 +200,20 @@ protected override void OnDetachingFrom(VisualElement bindable)
 			base.OnDetachingFrom(bindable);
 		}
 
-		protected override void OnViewPropertyChanged(object sender, PropertyChangedEventArgs e)
+		protected override async void OnViewPropertyChanged(object? sender, PropertyChangedEventArgs e)
 		{
 			base.OnViewPropertyChanged(sender, e);
 			if (e.PropertyName == VisualElement.IsFocusedProperty.PropertyName)
 			{
-				currentStatus = View.IsFocused
+				currentStatus = View?.IsFocused is true
 					? ValidationFlags.ValidateOnFocusing
 					: ValidationFlags.ValidateOnUnfocusing;
-				_ = UpdateStateAsync(false);
+				await UpdateStateAsync(false);
```

Comment:
Any reason to convert this method to "async void" with allocation resources for awaiting the task result, which we don't need?

Reply:
Yes, every `Task` should be awaited. `_ = Task...` is a huge code smell.